enterprise/backend/src/metabase_enterprise/data_editing/configure.clj

```diff
@@ -46,15 +64,30 @@
                                :table.row/delete (isa? (:semantic_type field) :type/PK))]
                    {:id               (format "field-%s" (:name field))
                     :sourceType       "ask-user"
-                    :sourceTypeTarget (:name field)})}))
+                    :sourceValueTarget (:name field)})}))
 
-(mu/defn configuration :- ::action-configuration
+(mu/defn configuration :- [:or ::action-configuration [:map [:status ms/PositiveInt]]]
   "Returns configuration needed for a given action."
-  [{:keys [action-id action-kw] :as unified}
+  [{:keys [action-id action-kw] :as action}
    scope]
-  (cond
-    (pos-int? action-id)
-    (configuration-for-saved-action (:action-id unified))
+  (if (false? (:configurable action))
+    {:status 400, :body "Cannot configure this action"}
+    (cond
+      ;; Eventually will be put inside a nicely typed :configuration key
+      (:param-map action)
```

Comment:
Basically I think we want a nice `:configuration` map on our "unified" actions, which will hold this parameter config (which will no longer be a map), as well as the title, and other configurable stuff.

I think we'll leave config like whether the action is a row or a header action in the dashcard viz settings.

Whether the action is enabled - should that come in? I think not, cos maybe in future we'll let you refer to the same custom action from two Editables, and disable it independently.